pages/e/[eventId]/edita/index.tsx

```
@@ -393,34 +489,25 @@ export default function Edita({ event }) {
+  params: { eventId: string };
+}) {
+  try {
+    const res = await fetch(
+      `${process.env.NEXT_PUBLIC_API_URL}/events/${params.eventId}`
+    );
+    const event: EventDetailResponseDTO = await res.json();
+    if (!event) {
+      return { notFound: true };
+    }
+    return { props: { event } };
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)